services/QuillLMS/spec/models/concept_result_spec.rb

```diff
@@ -249,45 +215,17 @@
           "questionScore": 0.8
         }
       end
-      let(:old_concept_result) { create(:sentence_combining, concept: concept, activity_session: activity_session, concept_uid: concept.uid, metadata: metadata, activity_classification_id: activity.activity_classification_id) }
+      let(:concept_result) { ConceptResult.create_from_json({concept_id: concept.id, activity_session_id: activity_session.id, metadata: metadata, activity_classification_id: activity.activity_classification_id, question_type: 'sentence-combining'}) }
 
       it 'should return data in the same shape as a ConceptResult' do
```

Comment:
I think we want this test name to reference "should return data in the same shape as a legacy ConceptResult'"